CondFormats/JetMETObjects/src/JetCorrectionUncertainty.cc

```diff
@@ -69,23 +68,19 @@ JetCorrectionUncertainty::JetCorrectionUncertainty(const JetCorrectorParameters&
   mIsLepPyset = false;
   mIsLepPzset = false;
   mAddLepToJet = false;
-  mUncertainty = new SimpleJetCorrectionUncertainty(fParameters);
 }
 /////////////////////////////////////////////////////////////////////////
-JetCorrectionUncertainty::~JetCorrectionUncertainty() { delete mUncertainty; }
-/////////////////////////////////////////////////////////////////////////
 void JetCorrectionUncertainty::setParameters(const std::string& fDataFile) {
   //---- delete the mParameters pointer before setting the new address ---
```

Comment:
This comment is not needed any more, and it can be misleading
```suggestion
```

Reply:
removed